dj_debexpo/dj_debexpo/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+urlpatterns = patterns('',
+    # Examples:
+    # url(r'^$', 'dj_debexpo.views.home', name='home'),
+    # url(r'^blog/', include('blog.urls')),
+
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
Oh, huh! I'm kind of excited to hypothetically use the Django admin UI, rather than SSH-ing in to the machine that runs mentors.debian.net, in the future.